src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,64 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ELEMENTS = 10;
+    private K[] keys;
+    private V[] values;
+    private K key;
+    private V value;
+    private int sizeStorage = 0;
```

Comment:
```suggestion
    private int size;
```
0 is default for int, let's keep name simple

Reply:
not fixed